portal/package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@ministryofjustice/frontend": "1.8.0",
+    "@ukef/dtfs2-common": "^1.0.0",
```

Comment:
Added the `auditDetails` helpers when sending feedback from tfm & portal ui. Since they're both using the open router, the user isn't picked up automatically so it made sense to re-use these for `no user logged in` vs `portal` users